### PR TITLE
CRM-17309 Classify no MX record bounce pattern as a DNS Bounce

### PR DESCRIPTION
**Pull request by seamuslee001**
- [CRM-17309: Classify No MX record bounce as DNS Bounce type](https://issues.civicrm.org/jira/browse/CRM-17309)

### CODE REVIEW

**Comment by seamuslee001** [comment]
This is a rebase of PR https://github.com/civicrm/civicrm-core/pull/6854. @davecivicrm Dave I think this should be fine to go but you may want to double check with CiviSMTP that this works with them

**Comment by seamuslee001** [comment]
Jenkins re test this please

**Comment by seamuslee001** [comment]
jenkins re-test this please

File: xml/templates/civicrm_data.tpl [hunk 1]
```diff
@@ -1239,6 +1239,7 @@ INSERT INTO civicrm_mailing_bounce_pattern
     (@bounceTypeID, 'no (mail server|matches to nameserver query|dns entries)'),
     (@bounceTypeID, 'reverse dns entry'),
     (@bounceTypeID, 'Host or domain name not found');
```
Comment (davecivicrm) [inline xml/templates/civicrm_data.tpl]: @seamuslee001  Line ending needs to be changed to a comma on line 1241.

**Comment by davecivicrm** [comment]
Pretty sure there's a fatal syntax error on line 1241 (semi-colon where there should be a comma).

**Comment by seamuslee001** [comment]
@davecivicrm I think your right, have changed it now

Seamus

**Comment by seamuslee001** [comment]
Jenkins re test this please

**Comment by seamuslee001** [comment]
Jenkins re test this please

**Pull request merged**